bin/test-workflow/4_admin_cluster_prep.sh

```diff
@@ -4,6 +4,8 @@ set -euo pipefail
 cd "$(dirname "$0")" || ( echo "cannot cd into dir" && exit 1 )
 
 TIMEOUT="${TIMEOUT:-5m0s}"
+TEST_CLIENT_IMAGE_TAG="${TEST_CLIENT_IMAGE_TAG:-edge}"
+echo "***TEMP*** env: TEST_CLIENT_IMAGE_TAG: $TEST_CLIENT_IMAGE_TAG"
```

Comment:
Was this intended to stay in?

Reply:
No, this is some temporary debug code. I think I figured out what was missing, so I'll remove this now.